test/tools/ossfuzz/yulProto_diff_ossfuzz.cpp

```diff
@@ -76,6 +77,7 @@ DEFINE_PROTO_FUZZER(Function const& _input)
 	{
 		yulFuzzerUtil::interpret(
 			os1,
+			*EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion()),
```

Comment:
and here

Reply:
and please change the interpret call further down in the same source file accordingly.